building/configlet/lint.md

```diff
@@ -292,3 +292,7 @@ The `config.json` file should have the following checks:
 - The `"contributors"` values must not have duplicates
 - The `"contributors"` values are treated case-insensitively
 - Users can only be listed in either the `"authors"` or `"contributors"` array (no overlap)
+
+## Glossary
+
+- _kebab_case_: strings containing only characters in the range `[a-z0-9-]`
```

Comment:
```suggestion
- _kebab_case_: strings containing only characters in the range `[a-z0-9-]`; the first character must be in the range `[a-z0-9]` (no dash)

Reply:
@ee7 This clear?

Reply:
Depends on whether you want `hello------------erik` and `hello-` to be valid slugs :)

Reply:
I think we should go with the regexp`^([a-z0-9][a-z0-9]*)(-[a-z0-9]+)*$`

Reply:
```suggestion
- _kebab_case_: strings starting and ending with a character in the range `[a-z0-9]`, with zero or more characters in between in the range `[a-z0-9-]`. In regular expression format: `^[a-z0-9]+(-[a-z0-9]+)*$`
```

Reply:
@iHiD @ee7 Something like this?

Reply:
```suggestion
- _kebab_case_: strings containing only characters in the range `[a-z0-9-]` optionally seperated by dashes (e.g. "two-fer"). In regular expression format: `^[a-z0-9]+(-[a-z0-9]+)*$`
```